### PR TITLE
Add status route

### PR DESCRIPTION
**Pull request by vincentvanderweele**
Added status route:

```bash
$ curl localhost:9999/status
{"status": "health-check-ok"}
```

### CODE REVIEW

**Comment by vincentvanderweele** [comment]
Retest this please

**Comment by vincentvanderweele** [comment]
Retest this please

**Comment by vincentvanderweele** [comment]
Retest this please

File: main.go [hunk 1]
```diff
@@ -34,9 +35,13 @@ func main() {
 		exit(nil, err)
 	}
 
-	handlerChain := logging.NewLogMiddleware(h)
+	http.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
```
Comment (heppu) [inline main.go]: I think it's bit cleaner to marshal JSON even though it couple lines more code:
```go
package main

...

var healtOK, _ = json.Marshal(&struct {
	Status string `json:"status"`
}{Status: "health-check-ok"})

func main() {

      ...

	http.HandleFunc("/status", func(w http.ResponseWriter, _ *http.Request) {
		fmt.Fprint(w, healtOK)
	})

      ...

}
```

Comment (vincentvanderweele) [inline main.go]: I was afraid I wouldn't get away with this :D

Comment (ljanne) [inline main.go]: :D

**Comment by vincentvanderweele** [comment]
Retest this please

**Comment by vincentvanderweele** [comment]
Retest this please

**Pull request merged**